File: src/main/java/edu/kit/iti/formal/stvs/model/examples/PedestrianLightExample.java
package edu.kit.iti.formal.stvs.model.examples;

import java.net.URL;

/**
 * @author Alexander Weigl
 * @version 1 (01.04.17)
 */
public final class PedestrianLightExample extends Example {
    public PedestrianLightExample() {
        name = "Pedestrian Light";
        description = "Demo";
        htmlHelp = getClass().getResource("light.html").toExternalForm();//"https://formal.iti.kit.edu/stvs/minmax.html";
        sessionFile = getClass().getResource("light.xml");
    }
}

File: stvs/src/main/kotlin/edu/kit/iti/formal/stvs/model/examples/PedestrianLightExample.kt
package edu.kit.iti.formal.stvs.model.examples

/**
 * @author Alexander Weigl
 * @version 1 (01.04.17)
 */
class PedestrianLightExample : Example() {
    init {
        name = "Pedestrian Light"
        description = "Demo"
        htmlHelp = javaClass.getResource("light.html").toExternalForm() //"https://formal.iti.kit.edu/stvs/minmax.html";
        sessionFile = javaClass.getResource("light.xml")
    }
}
